charts/linkerd-control-plane/values.yaml

```diff
@@ -198,6 +198,14 @@ proxy:
   # "all-unauthenticated", "cluster-authenticated", "cluster-unauthenticated", "deny"
   # @default -- "all-unauthenticated"
   defaultInboundPolicy: "all-unauthenticated"
+  # -- LivenessProbe timeout and delay configuration
+  livenessProbe:
+    initialDelaySeconds: 10
```

Comment:
So the default here used to be zero? Setting the new default to 10 is going to significantly delay the startup time for pods. How about leaving the default as it was? Pathological cases would still have an escape hatch with this new setting.

Reply:
it was hardcoded in to 10, see the _proxy.tpl change. The default did not change.

Reply:
:blush: